skyportal/handlers/api/allocation.py

```diff
@@ -640,19 +640,24 @@ def put(self, allocation_id):
 
             data = self.get_json()
             data["id"] = allocation_id
+
+            only_update_changed_fields = data.pop("only_update_changed_fields", False)
```

Comment:
```suggestion
            only_update_changed_fields = data.pop("only_update_modified_fields", False)
```

Let's try to keep using the same vocabulary as the rest of the app. One might wonder if it's not easier to just have a `replace_altdata` True/False here instead. That's just a suggestion / personal preference, doesn't mean that you should make that change